paimon-common/src/main/java/org/apache/paimon/utils/RoaringBitmap32.java

```diff
@@ -89,6 +91,10 @@ public void deserialize(DataInput in) throws IOException {
         roaringBitmap.deserialize(in);
     }
 
+    public void deserialize(DataInput in, @Nullable byte[] buffer) throws IOException {
+        roaringBitmap.deserialize(in, buffer);
```

Comment:
if `deserialize(in, null)` is better, can you just modify `RoaringBitmap32.deserialize(DataInput)`?

Reply:
I totally agree with this, but be aware that it may take up 8KB of memory during deserialization, although I think it's reasonable.